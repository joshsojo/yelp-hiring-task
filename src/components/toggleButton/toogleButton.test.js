import { shallow } from "enzyme";
import React from "react";
import configureStore from "redux-mock-store";
import { Provider } from "react-redux";
import ToggleButton from "./toggleButton.component";

describe("Google Map Container Component", () => {
  const mockStore = configureStore();
  let wrapper;
  beforeEach(() => {
    const mockProps = mockStore({
      toggleMap: jest.fn()
    });
    wrapper = shallow(
      <Provider store={mockProps}>
        <ToggleButton />
      </Provider>
    );
  });
  test("Renders without crashing", () => {
    expect(shallow(<wrapper />)).toMatchSnapshot();
  });
});
