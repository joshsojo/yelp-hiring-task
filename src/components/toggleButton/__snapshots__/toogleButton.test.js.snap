// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Google Map Container Component Renders without crashing 1`] = `ShallowWrapper {}`;
